pkg/promclient/promclient.go

```diff
@@ -149,6 +149,21 @@ func (c *Client) req2xx(ctx context.Context, u *url.URL, method string, headers
 func IsWALDirAccessible(dir string) error {
 	const errMsg = "WAL dir is not accessible. Is this dir a TSDB directory? If yes it is shared with TSDB?"
 
+	f, err := os.Stat(filepath.Join(dir, "snapshots"))
```

Comment:
Umm, is it inteded to check `snapshots` dir under `IsWALDirAccessible` and checking `wal` under `IsSnapShotDirAccessible`

Reply:
Ups 😄 
Fixed.